google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/MessageDispatcher.java

```diff
@@ -350,6 +354,32 @@ private OutstandingMessage(ReceivedMessage receivedMessage, AckHandler ackHandle
     }
   }
 
+  private static class ReceiptCompleteData {
+    private OutstandingMessage outstandingMessage;
+    private Boolean receiptComplete;
+
+    private ReceiptCompleteData(OutstandingMessage outstandingMessage, Boolean receiptComplete) {
+      this.outstandingMessage = outstandingMessage;
+      this.receiptComplete = receiptComplete;
```

Comment:
This should never be instantiated with receiptComplete as true. So don't take in receiptComplete and instead just initialize the value to false always.